tests/dbshell/test_oracle.py

```diff
@@ -0,0 +1,70 @@
+import os
+import signal
+from unittest import mock
+
+from django.db.backends.oracle.client import DatabaseClient
+from django.test import SimpleTestCase
+
+
+class MockConnection(object):
```

Comment:
Do we need to mock connection? I think it is not necessary, you can use `django.db.connection`.